### ISSUE TITLE
Starting module interception failed

### ISSUE THREAD

**Issue by Morasithil** [safing/portmaster#1354]
**Pre-Submit Checklist**:

- Check applicable sources for existing issues:
  - [Windows Known Issues](https://docs.safing.io/portmaster/install/windows#known-issues)
  - [Linux Known Issues](https://docs.safing.io/portmaster/install/linux#compatibility)
  - [Github Issues](https://github.com/safing/portmaster/issues?q=is%3Aissue+label%3Abug)

**What happened**:
Starting module interception failed

**What did you expect to happen?**:

**How did you reproduce it?**:
Been rebooting several times, the error remains the same.
Windows 10 22H2

**Debug Information**:

**Comment by Morasithil** [comment]
[2023-10-29-14-42-48.error.log](https://github.com/safing/portmaster/files/13198407/2023-10-29-14-42-48.error.log)
[2023-10-29-14-42-48.log](https://github.com/safing/portmaster/files/13198409/2023-10-29-14-42-48.log)
[2023-10-29-14-44-42.error.log](https://github.com/safing/portmaster/files/13198410/2023-10-29-14-44-42.error.log)
[2023-10-29-14-44-42.log](https://github.com/safing/portmaster/files/13198411/2023-10-29-14-44-42.log)
[2023-10-29-14-46-06.error.log](https://github.com/safing/portmaster/files/13198412/2023-10-29-14-46-06.error.log)
[2023-10-29-14-46-06.log](https://github.com/safing/portmaster/files/13198413/2023-10-29-14-46-06.log)
[2023-10-29-14-50-50.error.log](https://github.com/safing/portmaster/files/13198414/2023-10-29-14-50-50.error.log)
[2023-10-29-14-50-50.log](https://github.com/safing/portmaster/files/13198415/2023-10-29-14-50-50.log)
[2023-10-29-15-02-06.error.log](https://github.com/safing/portmaster/files/13198416/2023-10-29-15-02-06.error.log)
[2023-10-29-15-02-06.log](https://github.com/safing/portmaster/files/13198417/2023-10-29-15-02-06.log)
[2023-10-29-14-41-22.error.log](https://github.com/safing/portmaster/files/13198418/2023-10-29-14-41-22.error.log)
[2023-10-29-14-41-22.log](https://github.com/safing/portmaster/files/13198419/2023-10-29-14-41-22.log)
[2023-10-29-14-42-35.error.log](https://github.com/safing/portmaster/files/13198420/2023-10-29-14-42-35.error.log)
[2023-10-29-14-42-35.log](https://github.com/safing/portmaster/files/13198421/2023-10-29-14-42-35.log)
[2023-10-29-14-46-05.log](https://github.com/safing/portmaster/files/13198422/2023-10-29-14-46-05.log)
[2023-10-29-14-47-14.log](https://github.com/safing/portmaster/files/13198423/2023-10-29-14-47-14.log)
[2023-10-29-14-50-50.log](https://github.com/safing/portmaster/files/13198424/2023-10-29-14-50-50.log)
[2023-10-29-14-50-59.log](https://github.com/safing/portmaster/files/13198426/2023-10-29-14-50-59.log)
[2023-10-29-15-02-06.log](https://github.com/safing/portmaster/files/13198427/2023-10-29-15-02-06.log)
[2023-10-29-15-02-32.log](https://github.com/safing/portmaster/files/13198428/2023-10-29-15-02-32.log)
[2023-10-29-14-41-21.log](https://github.com/safing/portmaster/files/13198429/2023-10-29-14-41-21.log)
[2023-10-29-14-41-28.log](https://github.com/safing/portmaster/files/13198430/2023-10-29-14-41-28.log)
[2023-10-29-14-42-35.log](https://github.com/safing/portmaster/files/13198431/2023-10-29-14-42-35.log)
[2023-10-29-14-42-43.log](https://github.com/safing/portmaster/files/13198432/2023-10-29-14-42-43.log)
[2023-10-29-14-42-48.log](https://github.com/safing/portmaster/files/13198433/2023-10-29-14-42-48.log)
[2023-10-29-14-44-42.log](https://github.com/safing/portmaster/files/13198434/2023-10-29-14-44-42.log)
[2023-10-29-14-45-05.log](https://github.com/safing/portmaster/files/13198435/2023-10-29-14-45-05.log)

**Comment by Morasithil** [comment]
Any help with the error "Starting module interception failed" ?

**Comment by Raphty** [comment]
IT looks like you have issue with your installation. did you try a reinstall?

**Comment by Raphty** [comment]
Was this right after install?
Do you have Antivirus installed?
Did you make significant changes to the OS? or are you running it on default?

**Comment by Morasithil** [comment]
Yes right after install.
Yes windows defender but this should not be an issue.
Yes i made significant changes to the OS.

I've found a solution, but it's not optimal.
Installing protonvpn fixed the problem, i asume that whatever happens during the protonvpn installing "repaired" whatever my significant changes broke. But note that the internet connection was working, i've tested multiple programs like simplewall and the librewolf browser.
However, i've also installed mullvadvpn which failed to connect.
So portmaster and mullvadvpn are the two programs that refused to work, but all that was solved with installing protonvpn.
I know safing is absolutely not responsible for any of this, but i'd like to ask if you have any idea what might have been the issue?
Could share the script i used to change the OS, you would be able to reproduce the issue.
ProtonVPN installed .NET Framework 4.7.2, but shouldn't that components be preinstalled in windows?

**Comment by Raphty** [comment]
sorry, I don't know of any good scripts for that sort of thing. maybe someone from the community can help.

in regards to VPN compatibility there is a section in the wiki: https://wiki.safing.io/en/Portmaster/App/Compatibility

**Comment by livan3li** [comment]
hi, @Raphty I'm getting same error right now.
OS : Debian 12.2.0( not a Linux distro fork, Debian itself)
1.First I've installed the application. After installation, I've opened the App and I've disabled all filters in first configuration window.
2. After some time, i removed the app and reinstalled back. And this time the app didnt ask me for configuraiton.
3. Now in dashboard it gives the error in the title. 
4. and no application, no connection is being observed
![image](https://github.com/safing/portmaster/assets/46044710/8458cb7d-a16f-47c9-830e-4fe08a5f3750)
